management-gateway/src/main/java/org/frankframework/management/gateway/HazelcastOutboundGateway.java

```diff
@@ -60,6 +60,9 @@ public class HazelcastOutboundGateway implements InitializingBean, ApplicationCo
 	private final String requestTopicName = HazelcastConfig.REQUEST_TOPIC_NAME;
 	private ITopic<Message<?>> requestTopic;
 
+	@Autowired
+	private JwtKeyGenerator jwtGenerator;
```

Comment:
Ik mag het hieronder niet toevoegen op regel 91, maar er zit een typo in `silentylyRemoveQueue`